homeassistant/components/sensor/simulated.py

```diff
@@ -49,6 +52,7 @@
     vol.Optional(CONF_PHASE, default=DEFAULT_PHASE): vol.Coerce(float),
     vol.Optional(CONF_FWHM, default=DEFAULT_FWHM): vol.Coerce(float),
     vol.Optional(CONF_SEED, default=DEFAULT_SEED): cv.positive_int,
+    vol.Optional(CONF_ABSOLUTE_TIME, default=DEFAULT_ABSOLUTE_TIME): cv.boolean
```

Comment:
I don't think that absolute time is the correct description. It's a description of the value, not what the value is used for.

Reply:
What do you propose?